src/components/RecipientCard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <SmallPage class="recipient-card">
+        <Identicon :address="address"></Identicon>
+        <AddressDisplay :address="address"></AddressDisplay>
+        <div class="seperator"></div>
+        <input class="nq-input vanishing" type="text" placeholder="Add a Message"/>
+        <div class="nq-text-s">(optional)</div>
+    </SmallPage>
+</template>
+
+
+<script lang="ts">
+    import '@nimiq/style/nimiq-style.min.css';
```

Comment:
We already import this in `App.vue`.
That's enough.
We don't specifically repeat this for each component.